src/datasets/datasets.v4.controller.ts

```diff
@@ -396,6 +377,7 @@ export class DatasetsV4Controller {
 
   // GET /fullfacets
   @UseGuards(PoliciesGuard)
+  // TODO: Check if this should be closed or oppened endpoint for public!?
```

Comment:
Fullfact is currently used in two places, Filter UI dropdown and datasets total.
I'd say it'd be easier to maintain frontend by including fullfacet endpoint for public 
<img width="261" alt="image" src="https://github.com/user-attachments/assets/d0371b61-9419-4452-816b-c8efae180170" />
<img width="495" alt="image" src="https://github.com/user-attachments/assets/23db702d-b863-4ee1-a752-669788364f2d" />